packages/fast-components-react-msft/src/pivot/examples.data.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import { Pivot, PivotHandledProps, PivotProps } from "./index";
+import Documentation from "./.tmp/documentation";
+import { TabsItem } from "@microsoft/fast-components-react-base";
+import { uniqueId } from "lodash-es";
+import schema from "./pivot.schema.json";
+import pivotItemContentSchema from "../../app/components/pivot-item-content.schema.json";
+import pivotItemTabSchema from "../../app/components/pivot-item-tab.schema.json";
+
+const pivotItem1: TabsItem = {
+    tab: {
+        id: pivotItemTabSchema.id,
+        props: {
+            children: "pivot one",
+        },
+    } as any,
+    content: {
+        id: pivotItemContentSchema.id,
+        props: {
+            children: "pivot one content",
+        },
+    } as any,

```

Comment:
@eljefe223 is there a reason we are coercing prop types here? Last time I saw this it was due to an actual bug.

Reply:
Looks like this is due to how we are manipulating children